dask_mongo/core.py

```diff
@@ -10,14 +10,18 @@
 from dask.base import tokenize
 from dask.graph_manipulation import checkpoint
 
+import dask_mongo
+
+app_info = f"dask-mongo/{dask_mongo.__version__}"
```

Comment:
This line is also causing a circle import error because `dask_mongo/__init__.py` import code from `dask_mongo/core.py`, which itself attempts to `import dask_mongo`. I'd recommend storing the version string in a separate `dask_mongo/_version.py` file and then `import`ing it into `dask_mongo/__init__.py` and `dask_mongo/core.py` separately

Reply:
Thanks for the suggestion @jrbourbeau, CI is looking green now. If you don't have any more comments, I'll let you hit the green button.